### PR TITLE
Address a recent problem in our release process

### PR DESCRIPTION
**Pull request by fpesce**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[x] Security
[x] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/blog/2019/incorrect-sidecar-image-1.2.4/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Istio 1.2.4 sidecar image vulnerability
+description: An erroneus 1.2.4 sidecar image was available due to a faulty release operation.
+publishdate: 2019-09-10
+attribution: The Istio Team
+keywords: [community,blog,security]
+---
+To the Istio’s user community,
+
+**TL;DR**: For the period between Aug 23rd 2019 09:16PM PST and Sep 6th 2019 09:26AM PST a Docker image shipped as Istio `proxyv2` 1.2.4 (c.f. [https://hub.docker.com/r/istio/proxyv2](https://hub.docker.com/r/istio/proxyv2) ) contained a faulty version of the proxy against the security bugs [ISTIO-SECURITY-2019-003 and ISTIO-SECURITY-2019-004](/blog/2019/istio-security-003-004/).
```
Comment (geeknoid) [inline content/en/blog/2019/incorrect-sidecar-image-1.2.4/index.md]: Remove the `**&TL;DR**: `, it's not necessary.

File: content/en/blog/2019/incorrect-sidecar-image-1.2.4/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+If you have installed Istio 1.2.4 during that time, please consider upgrading to Istio 1.2.5 that also contains additional security fixes.
+
+**Full**:
```
Comment (geeknoid) [inline content/en/blog/2019/incorrect-sidecar-image-1.2.4/index.md]: Replace `**Full**:" with `## Detailed explanation`

File: content/en/blog/2019/incorrect-sidecar-image-1.2.4/index.md (hunk 1 above)
Comment (fpesce) [reply content/en/blog/2019/incorrect-sidecar-image-1.2.4/index.md]:
```suggestion
For the period between Aug 23rd 2019 09:16PM PST and Sep 6th 2019 09:26AM PST a Docker image shipped as Istio `proxyv2` 1.2.4 (c.f. [https://hub.docker.com/r/istio/proxyv2](https://hub.docker.com/r/istio/proxyv2) ) contained a faulty version of the proxy against the security bugs [ISTIO-SECURITY-2019-003 and ISTIO-SECURITY-2019-004](/blog/2019/istio-security-003-004/).
```

File: content/en/blog/2019/incorrect-sidecar-image-1.2.4/index.md (hunk 2 above)
Comment (fpesce) [reply content/en/blog/2019/incorrect-sidecar-image-1.2.4/index.md]:
```suggestion
## Detailed explanation

```

**Pull request merged**